src/web/wms/src/main/resources/GeoServerApplication_ca.properties

```diff
@@ -1,103 +1,104 @@
-#Generated by ResourceBundle Editor (http://eclipse-rbe.sourceforge.net)

```

Comment:
The entire file got modified... I believe it's newlines changes. Can you avoid it?